lib/src/voip/voip.dart

```diff
@@ -366,6 +373,11 @@ class VoIP {
             'Ignoring call reject for room $roomId claiming to be for call in room ${call.room.id}');
         return;
       }
+      if (call.remotePartyId != partyId) {
```

Comment:
remotePartyId is only set when there you already get a answer, if there was no answer, this would break ending calls

Reply:
added a null check before that